proposals/0253-static-subscripts.md

```diff
@@ -0,0 +1,132 @@
+# Static and class subscripts
+
+* Proposal: [SE-0253](0253-static-subscripts.md)
```

Comment:
Usually it makes sense to delay numbering a proposal until it's merged. E.g. there's already an [SE-0253](https://github.com/apple/swift-evolution/blob/master/proposals/0253-callable.md).

Reply:
Yes, I'm quickly learning that lesson. 😅

Reply:
No worries, I'll fix it